extra/benchmark/hashtables/hashtables.factor

```diff
@@ -7,7 +7,7 @@ QUALIFIED: assocs
 IN: benchmark.hashtables
 
 MEMO: strings ( -- str )
-    1 100 [a,b] 1 [ + ] accumulate nip [ number>string ] map ;
+    1 100 [a,b] 0 [ + ] accumulate* [ number>string ] map ;
```

Comment:
I think it's the same?
Le 8 mars 2016 15:41, "John Benediktsson" <notifications@github.com> a
écrit :

> In extra/benchmark/hashtables/hashtables.factor
> <https://github.com/factor/factor/pull/1548#discussion_r55367255>:
>
> > @@ -7,7 +7,7 @@ QUALIFIED: assocs
> >  IN: benchmark.hashtables
> >
> >  MEMO: strings ( -- str )
> > -    1 100 [a,b] 1 [ + ] accumulate nip [ number>string ] map ;
> > +    1 100 [a,b] 0 [ + ] accumulate* [ number>string ] map ;
>
> These aren't the same thing, though? (I know it's just a benchmark, but...)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/factor/factor/pull/1548/files#r55367255>.
>